examples/deferred_processing/README.md

```diff
@@ -0,0 +1,325 @@
+# Deferred Processing and Transactions
+
+This example, 
+which uses a simple DAG based on our [OpenCV example](https://github.com/pachyderm/pachyderm/tree/master/examples/opencv), 
+illustrates two Pachyderm usage patterns for fine-grain control over when pipelines trigger jobs.
+
+[Deferred processing](https://docs.pachyderm.com/latest/how-tos/deferred_processing/) 
+is a Pachyderm technique for controlling when data gets processed.
+Deferred processing uses branches to prevent pipelines from triggering on every input commit.
+
+[Transactions](https://docs.pachyderm.com/latest/how-tos/use-transactions-to-run-multiple-commands/) are a Pachyderm feature 
+that allows you to batch match multiple operations at once,
+such as committing data to two different repos, 
+but only trigger a single job, 
+so data from both repos gets processed together.
+
+## Prerequisites
+
+Before you begin, you need to have Pachyderm v1.9.8 or later installed on your computer or cloud platform. 
+See [Deploy Pachyderm](https://docs.pachyderm.com/latest/deploy-manage/deploy/).
+
+## Pipelines
+
+The following diagram demonstrates the DAG that is used in this example.
+
+![Example DAG](./example_dag.png)
```

Comment:
The image is not loading for some reason. Maybe try just `![Example DAG](example_dag.png)`

Reply:
I can see it in the file preview area, but that change won't hurt.